src/SourceLocator/Located/InternalLocatedSource.php

```diff
@@ -8,12 +8,19 @@
  */
 class InternalLocatedSource extends LocatedSource
 {
+
+    /**
+     * @var string
+     */
+    private $extensionName;
+
     /**
      * {@inheritDoc}
      */
-    public function __construct(string $source)
+    public function __construct(string $source, string $extensionName)
```

Comment:
BC break to be documented: you may consider waiting for the other patch to be merged though

Reply:
Yes, I will fix this after https://github.com/Roave/BetterReflection/pull/348 is merged.